FIRO_TSEnsembles/src/main/java/hec/EnsembleDatabase.java

```diff
@@ -0,0 +1,14 @@
+package hec;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+import hec.ensemble.*;
+
+public interface EnsembleDatabase extends AutoCloseable {
+    Ensemble getEnsemble(RecordIdentifier timeseriesID, ZonedDateTime issue_time);
```

Comment:
argument timeseriesID could become recordID

Reply:
adding in next commit